K8SPG-463 document private registry (#145)
percona/k8spg-docs: docs/custom-registry.md

# Use Docker images from a custom registry

> Using images from a private Docker registry may required for privacy, security

Reviewer: ```suggestion
Using images from a private Docker registry may be required for privacy, security
```

or other reasons. In these cases, Percona Operator for PostgreSQL allows the use

> of a custom registry This following example illustrates how this can be done by

Reviewer: ```suggestion
of a custom registry. The following example illustrates how this can be done by
```

the example of the Operator deployed in the OpenShift environment.

## Prerequisites 

1. First of all login to the OpenShift and create project.

    ``` {.bash data-prompt="$" }
    $ oc login
    Authentication required for https://192.168.1.100:8443 (openshift)
    Username: admin
    Password:
    Login successful.
    $ oc new-project pg
    Now using project "pg" on server "https://192.168.1.100:8443".
    ```

2. There are two things you will need to configure your custom registry access:

    * the token for your user,

    * your registry IP address.

>      The token can be find out with the following command:

Reviewer: ```suggestion
     The token can be found with the following command:
```
